commands/priority_test.go

```diff
@@ -49,3 +49,60 @@ func Test_getTemplateStoreURL(t *testing.T) {
 		})
 	}
 }
+
+func Test_getOverrideNamespace(t *testing.T) {
+	const defaultFunctionNamespace = defaultFunctionNamespace
+	failCases := 0
+
+	tests := []struct {
+		stack    string
+		flag     string
+		want     string
+		scenario string
+	}{
+		// Test cases
+		{
+			stack:    "",
+			flag:     "",
+			want:     defaultFunctionNamespace,
+			scenario: "no namespace value set in flag or in namespace field of stack file",
+		},
+
+		{
+			stack:    "openfaas-fn",
+			flag:     "default",
+			want:     "default",
```

Comment:
I did ask you to remove the text "default" since it isn't in @LucasRoesler testing grid and we have defaultFunctionNamespace as a default along with `openfaas-fn`.

It's all rather confusing.

If you don't want to use what was suggested, what about:

`dev`, `staging` and `openfaas-fn` (default)`

That would be far clearer for me as a maintainer.